src/Native/Common/pal_utilities.h

```diff
@@ -61,8 +61,8 @@ inline typename std::make_unsigned<T>::type UnsignedCast(T value)
  * MSDN blogs: http://blogs.msdn.com/b/the1/archive/2004/05/07/128242.aspx
  */
 template <typename T, size_t N>
-char ( &_ArraySizeHelper( T (&array)[N] ))[N];
-#define ARRAY_SIZE( array ) (sizeof( _ArraySizeHelper( array ) ))
+char(&_ArraySizeHelper(T(&array)[N]))[N];
```

Comment:
Nit: I'd suggest doing these formatting changes in a separate commit from the build script changes.